src/psyclone/tests/generator_test.py

```diff
@@ -147,23 +145,6 @@ def test_script_file_wrong_extension():
             "extension" in str(error.value))
 
 
-def test_script_file_not_found_relative():
-    '''Checks that handle_script() in generator.py raises the expected
```

Comment:
Ah, so we already supported searching the Python path for the script. I guess this is a consequence of the use of `__import__` but what happens if there is a file with the same name somewhere on the original Python path? Perhaps @rupertford can remember why we do things this way?

Reply:
We now prepend instead of append.